internal/validation/validation.go

```diff
@@ -23,6 +23,8 @@ type fieldInfo struct {
 	parent schema.NamedType
 }
 
+type bindings = map[string]interface{}
```

Comment:
Any reason to create a type alias (the code below is totally ok keeping maps as maps, i'd rather follow the code style of the repo)